build/test-integration.sh

```diff
@@ -55,4 +57,13 @@ dotnet test \
     $testCaseFilter \
     --test-adapter-path:. \
     --logger:"xunit;LogFilePath=$ARTIFACTS_DIR/testResults/$testProjectName$integrationTestPlatform.Tests.xml" \
-    -c $BUILD_CONFIGURATION
\ No newline at end of file
+    -c $BUILD_CONFIGURATION
+
+if [ "$testCaseFilter"=="db" ];then
+		echo "Running integration tests with no categories (tests missed to be tagged in their categories)..."
+		dotnet test \
+			--filter "category!=node&category!=python&category!=php&category!=dotnetcore&category!=db" \
```

Comment:
This list of categories can, and will, change. It should be moved to some central location (i.e. an enum of the possible categories)